app/models/report.rb

```diff
@@ -129,14 +129,6 @@ def self.cache_key(report)
   end
 
   def add_filter(name, options = {})
```

Comment:
I wonder if `options` here is truly unstructured date, or if it can be turned into keyword arguments. 🤔